internal/cmdutil/cmdutil.go

```diff
@@ -22,6 +22,45 @@ import (
 const WarnAuthMessage = "not authenticated yet. Please run 'pscale auth login' " +
 	"or create a service token with 'pscale service-token create'"
 
+type PasswordRole int
+
+const (
+	Reader PasswordRole = iota
+	Writer
+	ReadWriter
+	Administrator
```

Comment:
When we import these constants, they don't tell exactly what they are. especially if its `cmdutil.Reader` or `cmdutil.Writer`. They look similar to `io.Reader` or `io.Writer`, though it's something entirely different. 

What do you think of renaming these by adding a `Role` suffix? : 

```suggestion
	ReaderRole PasswordRole = iota
	WriterRole
	ReadWriterRole
	AdministratorRole
```

That way it would read as `cmdutil.ReaderRole` or `cmdutil.WriterRole`